internal/dag/dag.go

```diff
@@ -701,6 +701,44 @@ type SecureVirtualHost struct {
 
 	// JWTProviders specify how to verify JWTs.
 	JWTProviders []JWTProvider
+
+	// Tracing defines custom tags and the service to send trace data
+	Tracing TracingConfig
```

Comment:
I don't think this is being used anywhere, is it? Since tracing is a global setting, it's being directly configured on the Listener via the Listener xdscache, and doesn't need to be represented in the DAG. If that's accurate, then we can revert the additions to internal/dag.